CakePHP/Sniffs/WhiteSpace/EmptyLinesSniff.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * PHP Version 5
+ *
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://pear.php.net/package/PHP_CodeSniffer_CakePHP
+ * @since         CakePHP CodeSniffer 2.4.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+
+/**
+ * Ensures that not more than one blank line occurs
+ *
+ * @author Mark Scherer
+ * @license MIT
```

Comment:
That was copied from the source https://github.com/spryker/code-sniffer/blob/master/Spryker/Sniffs/Whitespace/EmptyLinesSniff.php